generators/client/templates/angular/src/main/webapp/app/shared/_shared-libs.module.ts

```diff
@@ -9,12 +9,13 @@ import { InfiniteScrollModule } from 'angular2-infinite-scroll';
 @NgModule({
     imports: [
         NgbModule.forRoot(),
-        NgJhipsterModule.forRoot({
+        // TODO uncomment the code below and make it works with AOT
+        NgJhipsterModule/*.forRoot({
```

Comment:
You are probably right. Then we need to see how we can fix that. Do you have any idea?  Else plz commit the usecase to reproduce the issue in some git repo and ill take a look when i have time